variables.tf

```diff
@@ -108,15 +108,15 @@ variable "lambda_timeout" {
 
 
 variable "default_tags" {
-  type = object({ team=string, application=string })
+  type = object({ team = string, application = string })
   default = {
-    team: "DR",
-    application: "disaster-recovery"
+    team : "DR",
+    application : "disaster-recovery"
   }
 }
 
 variable "buckets" {
-  type    = map(object({ name = string, type = string }))
+  type = map(object({ name = string, type = string }))
 }
 
```

Comment:
Add the new optional ORCA variables so they are bubbled up for the user along with type, description, and default values.

sqs_delay_time 
maximum_message_size 
sqs_message_retention_time